pkg/operation/botanist/component/vpnseedserver/vpn_seed_server.go

```diff
@@ -49,6 +53,8 @@ const (
 	DeploymentName = v1beta1constants.DeploymentNameVPNSeedServer
 	// ServiceName is the name of the vpn seed server service running internally on the control plane in seed.
 	ServiceName = DeploymentName
+	// GatewayPort is the port exposed by the istio ingress gateway for the vpn seed service
+	GatewayPort = 8132
```

Comment:
Shouldn't this rather be in the `istio` package and not here?

Reply:
Why? This port is some vpn-seed-server specific configuration.

Reply:
Is it? OK, then the doc string was misleading/confusing (at least for me, looking at all of this without being into the details).